### DISCUSSION TITLE
Syncing DataGrid scroll bars

### DISCUSSION THREAD

**Discussion by DarkMortainius** [AvaloniaUI/Avalonia#15569] [Q&A]
I'm working on a program where I need a window with two DataGrids side-by-side for a comparison view. I would like to tie the two scrollbars together so that when you scroll in one DataGrid, it also scrolls the other. I tried putting each of them inside a ScrollViewer and was able to tie those together, but that didn't work because it would only scroll to the bottom of the currently displayed view in the DataGrid, it didn't actually scroll the DataGrid. Any ideas or pointers on how to implement this?

Thanks!

**Comment by workgroupengineering** [comment]
You can got ispiration from
 https://github.com/sourcegit-scm/sourcegit/blob/master/src%2FViews%2FTextDiffView.axaml#L22

**Reply by DarkMortainius** [reply]
While interesting, I don't think that will work with a DataGrid, unless there is a way of directly access the scroll offset of the DataGrid.

**Reply by timunie** [reply]
if you know the row, one can call `myDataGrid.ScrollIntoView()`
